LPD-25885 - Warehouse Search doesn't function

victorcdp opened the pull request:
https://liferay.atlassian.net/browse/LPD-25885

The problem is that there was no search param in the getWarehousesPage method

Commit:
LPD-25885 Add search param in rest-openapi.yaml

Commit:
LPD-25885 autogenerated (buildRest)

Commit:
LPD-25885 add search param to getWarehousesPage

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/42cba1bc7d4a1694662b6a372c5e2753443e024e">42cba1bc7d4a1694662b6a372c5e2753443e024e</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/LPD-25885">LPD-25885</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/d9dc29d4c9097b8b63ffa8a295151280b7933e10">d9dc29d4c9097b8b63ffa8a295151280b7933e10</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/5474/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/5474/jenkins-report.html">here</a>.</h5><pre>format-source-files:
     [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: Incorrect commit message: Missing breaking change in commit messages when the major version bumps up: ./modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-inventory-api/bnd.bnd (SourceCheck:BNDBreakingChangeCommitMessageCheck)
     [java] 
     [java]   at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:469)
     [java]   at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:301)
[stopwatch] [run.batch.test.action: 1:37.319 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:497: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:650: Java returned: 1
      [get] Getting: http://test-1-29/job/test-portal-source-format/5474//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20240515105557585.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20240515105557585.txt
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null934000045.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[beanshell] Truncating errors in /opt/dev/projects/github/liferay-portal/portal-impl/test-results/TEST-JenkinsLogAssertTest.xml
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1916567091
[junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 208ms
[junitreport] Deleting: /tmp/null1916567091
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [echo] run.batch.test.tear.down.start.timestamp: 05-15-2024 10:56:00:369 PDT
[stopwatch] [run.batch.test.tear.down: 0.000 sec]</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/5474/">test-portal-source-format#5474</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/5474/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/168">brianikim#168</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2568992177">[master] ci:test:sf - victorcdp &gt; brianikim - PR#168 - 2024-05-15[10:52:54]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2568992177<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/21609/">publish-testray-report#21609</a><br/></div></div>

Commit:
LPD-25885 baseline

# breaking

## What modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseResourceImpl.java

Change method signature from:

public Page<Warehouse> getWarehousesPage(
                    Filter filter, Pagination pagination, Sort[] sorts)
            throws Exception {

To:

public Page<Warehouse> getWarehousesPage(
                    String search, Filter filter, Pagination pagination, Sort[] sorts)
            throws Exception {

## Why

Without the search param, the search function was not working in the frontend data-sets using this API

----

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/42cba1bc7d4a1694662b6a372c5e2753443e024e">42cba1bc7d4a1694662b6a372c5e2753443e024e</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/LPD-25885">LPD-25885</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/8fd42a50556d6c83202e591d78027644ef162bdf">8fd42a50556d6c83202e591d78027644ef162bdf</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/6768/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/6768/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/6768/">test-portal-source-format#6768</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/6768/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/168">brianikim#168</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2569004089">[master] ci:test:sf - victorcdp &gt; brianikim - PR#168 - 2024-05-15[11:16:03]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2569004089<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/25940/">publish-testray-report#25940</a><br/></div></div>

brianikim commented:
Hi @victorcdp ,

It looks good. You will have to create a playwright test for this.

You can send a PR to liferay-commerce once the playwright test is also completed.
